scraping/scraping/start_scripts/amazon_eu.py

```diff
@@ -0,0 +1,318 @@
+import json
+import pkgutil
+from collections import defaultdict
+from logging import getLogger
+from typing import List, Optional
+
+from core.domain import ConsumerLifestageType, GenderType, ProductCategory
+
+logger = getLogger(__name__)
+
+
+def read_json(path: str) -> list:
+    data = pkgutil.get_data("scraping", path)
+    assert data
+    return json.loads(data.decode("utf-8"))
```

Comment:
Please comment why we do need the `pkgutil.get_data` and consider not using separated function because afaik it is not reused.